common/api-endpoints.ts

```diff
@@ -18,6 +18,8 @@ export interface SearchProjectsParams {
   query?: string
   organizationId?: number | number[]
   page?: number
+  page_size?: number
+  category?: number
```

Comment:
```suggestion
  categoryId?: number
```

Altera o nome desse argumento para seguir o padrão da interface. Na linha 33 você transforma para o parametro da API.

```
      if (key === 'categoryId') {
        cleanQuery.category = String(params.categoryId)
        return
      }
```